app/assets/stylesheets/application.scss

```diff
@@ -253,3 +254,21 @@ a {
   font-size: 1.2rem;
   position: relative;
 }
+
+.blue-text {
+  color: $search-result;
+  display: inline-block;
+  font-style: oblique;
+  margin-bottom: 20px;
+}
+
+.search-button {
+  margin-top: -28px;
+  display: inline-block;
+  position: relative;
+  background: rgba( 0, 0, 0, 0.6)
+}
+
+.search-button:hover {
+  background: rgba( 0, 0, 0, 0.2)
```

Comment:
Expected 0 spaces between parentheses instead of 1<br>Declaration should be terminated by a semicolon